### PR TITLE
Introduce `select` test helper

### PR DESCRIPTION
**Pull request by Robdel12**
I figured I'd give adding the select helper we provide in [Emberx-select](https://github.com/thefrontside/emberx-select/blob/master/test-support/helpers/register-select-helper.js) to ember core a shot.

### CODE REVIEW

**Comment by taras** [comment]
Awesome!: )

**Comment by stefanpenner** [comment]
:+1:

File: packages/ember-testing/lib/helpers.js [hunk 1]
```diff
@@ -123,6 +123,27 @@ function uncheck(app, selector, context) {
   return app.testHelpers.wait();
 }
 
+function select(app, selector, ...texts) {
+  let $options = app.testHelpers.findWithAssert(`${selector} option`);
+  let type = $options.prop('type');
+
+  assert(
+    `To select '${selector}', the elment must be a select box`,
```
Comment (beatle) [inline packages/ember-testing/lib/helpers.js]: typo in the 'element'

File: FEATURES.md [hunk 2]
```diff
@@ -46,6 +46,32 @@ for a detailed explanation.
   });
   ```
 
+* `ember-testing-select-helper`
+
+  Add a `select` test helper.
+
+  Say you your handlebars file looks like this:
```
Comment (alexdiliberto) [inline FEATURES.md]: "Say your handlebars file looks like this:"

**Comment by rwjblue** [comment]
I like this, but I do not think we should introduce new helpers in current acceptance tests.  Can you add a comment to https://github.com/emberjs/rfcs/pull/119 about this? I think this is best served by a general purpose "select" testing addon, and that RFC should make it pretty trivial to allow this to work from an addon.

Thank you for the time you put into this, and I'm sorry for letting it linger so long. I look forward to better testing structure, and I hope we can work together to make this helper a reality...

**Pull request closed without merging**